src/providers/historical-telemetry-provider.js

```diff
@@ -27,61 +27,55 @@ import {
 } from '../utils.js';
 
 export default class YamcsHistoricalTelemetryProvider {
-    constructor(url, instance) {
+    constructor(url, instance, openmct) {
         this.url = url;
         this.instance = instance;
+        this.openmct = openmct;
     }
 
     supportsRequest(domainObject) {
         return domainObject.type.startsWith('yamcs.');
     }
 
     request(domainObject, options) {
+        let metadata = this.openmct.telemetry.getMetadata(domainObject);
+        let isImagery = metadata.valuesForHints(['range']).length !== 0;
```

Comment:
There is actually an `imagery` hint for image telemetry. The imagery view itself [actually uses it](https://github.com/nasa/openmct/blob/59946e89ef57ce32f3c2298f20a5cd6def3c2ec1/src/plugins/imagery/ImageryViewProvider.js#L7).

Reply:
ah man, I totally had that but had switched it out for testing locally on yamcs (no imagery locally). I'll revert that.